applications/luci-app-ipsec-vpnserver-manyusers/luasrc/model/cbi/ipsec-server/settings.lua

```diff
@@ -1,31 +1,33 @@
 local s = require "luci.sys"
 local net = require"luci.model.network".init()
 local ifaces = s.net:devices()
-local m, s, o
+local mp, s, o
+
 mp = Map("ipsec", translate("IPSec VPN Server"))
-mp.description = translate(
-                     "IPSec VPN connectivity using the native built-in VPN Client on iOS or Andriod (IKEv1 with PSK and Xauth)")
-mp.template = "ipsec-server/index"
+mp.description = translate("IPSec VPN connectivity using the native built-in VPN Client on iOS or Andriod (IKEv1 with PSK and Xauth)")
+
+mp.template = "ipsec-server/ipsec-server_index"
 
 s = mp:section(TypedSection, "service")
 s.anonymous = true
+
 o = s:option(DummyValue, "ipsec-server_status", translate("Current Condition"))
-o.template = "ipsec-server/status"
+o.template = "ipsec-server/ipsec-server_status"
+
 enabled = s:option(Flag, "enabled", translate("Enable"))
```

Comment:
All of these custom variables `enabled` `clientip` etc. are pointless. Replace them with `o`.